### PR TITLE
Add check for test path inside repo path.

### PR DESCRIPTION
**Pull request by dwsteele**

### CODE REVIEW

**Comment by turnstep** [comment]
This works well for me, and gives a good error message.

File: .cirrus.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ freebsd_12_task:
   install_script: pkg install -y bash git postgresql-libpqxx pkgconf libxml2 gmake perl5 libyaml p5-YAML-LibYAML rsync
 
   script:
-    - perl ${CIRRUS_WORKING_DIR}/test/test.pl --no-gen --make-cmd=gmake --vm=none --vm-max=2 --no-coverage --no-valgrind --module=command --test=backup
+    - cd .. && perl ${CIRRUS_WORKING_DIR}/test/test.pl --no-gen --make-cmd=gmake --vm=none --vm-max=2 --no-coverage --no-valgrind --module=command --test=backup
```
Comment (turnstep) [inline .cirrus.yml:21]: Worth doing a `cd -` at the end?

Comment (turnstep) [inline .cirrus.yml:21]: `--vm=none` would seem to make `--vm-max` a no-op?

**Review by turnstep — APPROVED** [review]

File: .cirrus.yml (hunk 1 above)
Comment (dwsteele) [reply .cirrus.yml:21]: Doesn't seem like it is needed.

Comment (dwsteele) [reply .cirrus.yml:21]: --vm-max will still run things in parallel even when --vm=none. The option names have gotten a bit creaky -- they date back to when all this was done with vagrant VMs.

**Pull request merged**